### PR TITLE
formatting and spelling fixes for extend doc

### PR DESCRIPTION
**Pull request by tannerjfco**
## The Problem:
The lists on this page weren't properly formatted, so instead of lists we have a big wall of text. Additionally, I seemed to have struggled spelling Additional/Additionally correctly.

## The Fix:
This fixes the formatting and spelling issues.

### CODE REVIEW

File: docs/users/customization-extendibility.md [hunk 1]
```diff
@@ -5,28 +5,41 @@ For most standard web applications, ddev provides everything you need to success
...
 
 ## Conventions for defining additonal services
 When defining additional services for your project, it is recommended to follow these conventions to ensure your service is handled by ddev the same way the default services are.
+
 - Containers should be follow the naming convention `local-[sitename]-[servicename]`
+
 - Containers should be provided the following labels:
+
   - `com.ddev.site-name: ${DDEV_SITENAME}`
```
Comment (rfay) [inline docs/users/customization-extendibility.md]: No objection actually, but why are we using backticks on these? Normally a backtick would indicate a shell command or shell command output.

Comment (tannerjfco) [inline docs/users/customization-extendibility.md]: Backticks indicate code, whether it is a single tick for inline code or 3 ticks for code blocks.

Comment (rfay) [inline docs/users/customization-extendibility.md]: I know what backticks are in markdown :) The question is how we should use them in our documentation. The bullet list here isn't really code but concepts. Like I say, not requesting changes, but we're using backticks a lot and it doesn't necessarily improve our communication.

Comment (beeradb) [inline docs/users/customization-extendibility.md]: I can see the argument both ways. This feels like example configuration. I do think we should probably decide where and how we use backticks and code blocks in docs and try to standardize usage, though.

This probably points at a higher level need to develop some sort of style guide for docs before too long.

Comment (tannerjfco) [inline docs/users/customization-extendibility.md]: I think its appropriate usage here, as it is example configuration. I do tend to use them a bit liberally, for things like file names, indicating user input, etc. So valid we should probably determine more consistent use for them.

**Pull request merged**